projects/packages/stub-generator/README.md

```diff
@@ -0,0 +1,102 @@
+# stub-generator
+
+Extract stubs for specific functions/classes/etc from a codebase.
+
+This is intended for situations where you want stubs for specific parts of a large code base.
+If you want to extract stubs for everything, something like [php-stubs/generator](https://packagist.org/packages/php-stubs/generator) might work better for you.
+
+## Usage
+
+This is a fairly simple command-line application:
+
+```
+vendor/bin/jetpack-stub-generator definition-file.php
```

Comment:
```suggestion
bin/jetpack-stub-generator definition-file.php
```

Reply:
Depending on if this will be packaged and included elsewhere, I guess.

Reply:
> Depending on if this will be packaged and included elsewhere, I guess.

That's the plan. The readme is written from the perspective of someone who did `composer require --dev automattic/jetpack-stub-generator` to install it.